src/main/resources/db/local/V99_3__created_action_plans.sql

```diff
@@ -0,0 +1,7 @@
+insert into action_plan (id, referral_id, number_of_sessions, created_by_id, created_at, submitted_by_id, submitted_at, approved_by_id, approved_at)
+values ('700754aa-d868-4347-9c0f-50690773014e', '81d754aa-d868-4347-9c0f-50690773014e', 3, '8751622134', '2020-12-07 18:02:01.599803+00', null, null, null, null);
+
+insert into action_plan_activity (id, action_plan_id, desired_outcome_id, description, created_at)
+values ('800754aa-d868-4347-9c0f-50690773014e', '700754aa-d868-4347-9c0f-50690773014e', '301ead30-30a4-4c7c-8296-2768abfb59b5', 'Identify any outstanding anger management barriers', '2020-12-07 18:02:02.599803+00'),
```

Comment:
`uuidgen` is your friend

Reply:
Like it but sadly I need the id in join fields which means after generation i need to know what the last generated value was for action_plan.id when inserting into action_plan_activity